### ISSUE TITLE
Write Support

### ISSUE THREAD

**Issue by dweidenfeld** [dweidenfeld/plexdrive#10]
The program should be able to
* create directories
* delete directories
* create files
* write to files
* delete files

**Comment by wibuf** [comment]
Would write support allow apps like sonarr/radarr to write to the gdrive mount?

**Comment by dweidenfeld** [comment]
Yes

**Comment by After-Shock** [comment]
Shut up and take my money 💰.

**Comment by dweidenfeld** [comment]
I excluded the delete function in a separate issue, because it should be easy to implement and is currently more necessary for me :smile:

**Comment by fabioneves** [comment]
* waiting for this *

**Comment by toomuchio** [comment]
@dweidenfeld Any chance of getting move/rename support any time soon?

**Comment by stardustrider** [comment]
I use a separate rclone mount for writing, and it works a treat!

**Comment by zenjabba** [comment]
I works under a union.fs file system, as long as the plexdrive mount is first and RO and the rclone mount is second and RW, things are all coming up Millhouse!

**Comment by RyanWor** [comment]
@zenjabba Would you care to share more about this setup? Is Plex pointed to the union.fs or the plexdrive mount? Would love to restore the delete functionality directly in Plex. 
@dweidenfeld I noticed you moved delete to it's own issue, but also that issue is closed now. Change of plans or was it already implemented and I just missed something? 
Thanks!!

**Comment by l3uddz** [comment]
it would be pretty awesome if this feature also came with its own local cache that you could specify size limit on before writing/movies files to google. e.g. we can set a flag, --local-max-size=100G and every 100G, it will begin the upload/move process and removes the file once it receives confirmation its been uploaded (shows up in the changes list)

**Comment by zenjabba** [comment]
If you just need delete functionality, you can just use plexdrive directly without the need for unionfs. Unionfs is only now needed for putting files in your google drive.

**Comment by l3uddz** [comment]
my idea was that with being able to specify a local size limit you wouldnt even need to use unionfs. 
plexdrive would write to a local folder, and when that size limit is reached it would begin moving the files to google, in effect making unionfs not needed unless you want more than 1 read-only folder in your union.

**Comment by curbro** [comment]
I'm waiting on this one, too.

**Comment by SuperGaco** [comment]
This would be so amazing, I would cry if/when this happens.

**Comment by fabioneves** [comment]
@zenjabba can you share your setup config, using plexdrive + rclone mount? thanks!

**Comment by bassrock** [comment]
This would be useful for the DVR functionality and for SubZero to download subtitles.

**Comment by bassrock** [comment]
@zenjabba Can you share instructions on the union.fs mount?

**Comment by hbakhtiyor** [comment]
@bassrock check out [the comment](https://github.com/ncw/rclone/issues/972#issuecomment-305928426)

**Comment by hardlick** [comment]
> This would be useful for the DVR functionality and for SubZero to download subtitles.

any solutions for this?
